src/mvba/mvba/mod.rs

```diff
@@ -83,6 +85,14 @@ impl Mvba {
         Ok(true)
     }
 
+    pub fn current_tag(&self) -> Tag {
```

Comment:
We don't need these function, since the name also causes confusion; Do we have `next_tag`? 
Can we simply use `self.tag`?

Reply:
Mvba doesn't have a stable proposer it's derived from `current_proposer()` which in turn derives it from `self.l`.

Reply:
@davidrusu, adding a Tag may cause confusion in the future. Consider long-term code maintenance.